OlinkAnalyze/vignettes/non-parametric_analysis.Rmd

```diff
@@ -0,0 +1,230 @@
+---
+title: "Olink® non-parametric analysis"
+author: "Olink DS team"
+date: "`r Sys.Date()`"
+output: 
+  html_vignette:
+    toc: true
+    toc_depth: 2
+vignette: >
+  %\VignetteIndexEntry{Olink® non-parametric analysis}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  fig.width = 6,
+  fig.height = 3,
+  fig.align = "center",
+  collapse = TRUE,
+  comment = "#>")
+options(tibble.print_min = 4L, tibble.print_max = 4L)
+```
+
+```{r, echo=FALSE, message=FALSE}
+library(OlinkAnalyze)
+library(dplyr)
+library(stringr)
+library(ggplot2)
+library(kableExtra)
+```
+
+## Introduction
+
+When particular assumptions about population distribution or data characteristic are not met, a non-parametric test could be considered to analyze the data. Unlike parametric tests, which assume that the data follows a specific distribution (usually the normal distribution) and have specific parameters, non-parametric tests do not make these strict assumptions and are often referred to as distribution-free tests. Non-parametric tests are useful when dealing with ordinal, nominal, or skewed data or when the sample size is small.
+
+Here are some key characteristics and applications of non-parametric tests:
+
+No Assumptions about Data Distribution: Non-parametric tests do not assume that the data follows a specific probability distribution. This makes them robust in situations where the data’s distribution is unknown or does not meet the assumptions of parametric tests.
```

Comment:
Thought this vignette there are alot of capital letters in words im not sure should be. "No Assumptions about Data Distribution" should probably be "No assumptions about data distribution". 
Is there a reason why capital ? otherwise please change this.

Reply:
I think it would be fine. Just want to highlight these categories.